core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/filebeat.yml

```diff
@@ -1,26 +1,50 @@
 ---
-- name: Filebeat | Get information about installed packages as facts
+- name: Filebeat as DaemonSet | Get information about helm releases
+  delegate_to: localhost
+  become: false
+  run_once: true
+  environment: { KUBECONFIG: "{{ vault_location }}/../kubeconfig" }
+  command: helm list
+  register: check_filebeat_release
+
+- name: Reinstall Filebeat as DaemonSet from default namespace to custom namespace for "k8s as cloud service"
+  when:
+    - specification.helm_chart_name in check_filebeat_release.stdout
```

Comment:
Isn't `specification.helm_chart_name in check_filebeat_release.stdout` too wide of a condition? :thinking: What if user deploys a chart with a name that completely contains `specification.helm_chart_name`? :monkey:

Reply:
True, to be honest i wasn't sure of the condition and I was waiting for this kind of review just to be sure. I will do it more strictly

Reply:
I suspect of you'd use `stdout_lines` that should make it work. :thinking:

Reply:
Changes added, now it looks good in my opinion. Please check.